### ISSUE TITLE
valign/valign-wrapper doesn't work properly (!)

### ISSUE THREAD

**Issue by NonameSLdev** [Dogfalo/materialize#4451]
### Description
`valign-wrapper` class is suppose to only align children with the `valign` class. What it actually does is switch itself into a vertical view along with all the elements in it.
A `valign-wrapper` with 1 element that has the `valign` class also doesn't do anything.
The problem will become more clean in the codepen.

### Screenshots / Codepen
http://codepen.io/NonameSL/pen/XMoroE

**Comment by DanielRuf** [comment]
@NonameSLdev the codepen link seems to be wrong.

The helper does this:
```CSS
// Positioning
.valign-wrapper {
  display: flex;
  align-items: center;

  .valign {
    display: block;
  }
}
```

**Comment by Dogfalo** [comment]
Please update the codepen

**Comment by NonameSLdev** [comment]
@Dogfalo Oops, updated

**Comment by DanielRuf** [comment]
@NonameSLdev

You did not add the mentioned code to the right class. I changed it.

http://codepen.io/anon/pen/peqoWV

The behaviour is totally normal. In a `valign-wrapper` should be just (one) `.valign` item(s) because the boxmodel is flexbox and so the immediate childs of `.valign-wrapper` are rendered as columns.

**Comment by DanielRuf** [comment]
I was wrong. Anything in `.valign-wrapper` is vertically centered. It works like it should work.

**Comment by NonameSLdev** [comment]
@DanielRuf Huh. I see. Perhaps we should rename it / redo the tutorial on the valign helper as it is unclear what it's purpose is (and it's still unclear to me what it's supposed to do... Set the direction of the elements or center an element vertically). In materializecss.com/helpers, it's specified like so:

> You can easily vertically center things by adding the class valign-wrapper to the container holding the items you want to vertically align.
And the purpose of `.valign` isn't clear to me aswell.

**Comment by Dogfalo** [comment]
Fixed in 5d87ab2f1541cec16ab1296281d61f69b5dc3d03
